### PR TITLE
Update meta description tags

### PR DESCRIPTION
**Pull request by josh-**
- Use html meta tag as well as Open Graph tags
- Update base site description tag
- Add tags specific to each city

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -13,7 +13,7 @@ timezone:      Australia/Melbourne
 
 # Site Meta
 url: http://unihack.net
-description: "UNIHACK is Melbourne's premier student hackathon, organised by WIRED Monash"
+description: "UNIHACK is Australia's premier student hackathon, based in Melbourne and Sydney. Students get 24 hours to design and build something awesome – an app, website, piece of hardware... anything goes. Solve a problem, impress us and win some prizes!"
```
Comment (josh-) [inline _config.yml]: Open to discussion about this

File: _includes/head.html [hunk 2]
```diff
@@ -12,8 +17,8 @@
   {% else %}
   <meta property="og:image" content="{{ site.url }}/img/unihack-og.jpg" />
   {% endif %}
-  {% if page.og_description %}
-  <meta property="og:description" content="{{ page.og_description }}" />
+  {% if page.description %}
```
Comment (erfanio) [inline _includes/head.html]: This can be merged with the if statement above

File: melbourne/index.html [hunk 3]
```diff
@@ -9,6 +9,7 @@
 header_link: /melbourne
 title_override: true
 title: UNIHACK Melbourne &mdash; August 5-6, 2017
+description: UNIHACK is Melbourne's premier student hackathon, organised by WIRED Monash.
```
Comment (erfanio) [inline melbourne/index.html]: I think we should include the long description here as well

File: _includes/head.html (hunk 2 above)
Comment (josh-) [inline _includes/head.html]: Yeah sounds good 👍

File: melbourne/index.html (hunk 3 above)
Comment (josh-) [inline melbourne/index.html]: Something like

> "UNIHACK is Melbourne's premier student hackathon, running in August at Monash Business School. Students get 24 hours to design and build something awesome – an app, website, piece of hardware... anything goes. Solve a problem, impress us and win some prizes!"

?

Comment (erfanio) [inline melbourne/index.html]: yah 👍

File: _config.yml (hunk 1 above)
Comment (terencehuynh) [inline _config.yml]: Fine with it.

**Pull request merged**